src/components/LinksSection.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types";
+
+const LinksSection = ({ className, linksList }) => {
+  return (
+    <div className={className}>
+      <h3 className="pb-10 text-2xl font-bold uppercase font-heading">Links</h3>
+      <div className="flex flex-col gap-5 justify-center items-center w-full lg:flex-row lg:gap-10">
+        {linksList.map(({ title, target }) => (
+          <a
+            className="flex items-center justify-center py-2 px-5 rounded-lg font-heading font-normal text-center border-4 border-orange-600 shadow-md hover:bg-orange-600 hover:border-orange-600 w-full text-nowrap"
+            href={title}
```

Comment:
this is still not fixed

Reply:
what is not fixed? it's not a ` anymore. i thought that was the issue here?

Reply:
oh yeah it's fixed here, looks like I was looking at an old reference